addons/godot-xr-tools/audio/area_audio.gd

```diff
@@ -12,37 +10,37 @@ extends AudioStreamPlayer3D
 
 
 ## XRToolsAreaAudioType to associate with this Area Audio
-@export var area_audio_type : XRToolsAreaAudioType
+@export var area_audio_type: XRToolsAreaAudioType
 
-@onready var area : Area3D = get_parent()
+@onready var area: Area3D = get_parent()
 
 
-# Add support for is_class on XRTools classes
-func is_xr_class(xr_name:  String) -> bool:
+## Add support for is_class on XRTools classes
+func is_xr_class(xr_name: String) -> bool:
 	return xr_name == "XRToolsAreaAudio"
 
 
-# Called when the node enters the scene tree for the first time.
+## Called when the node enters the scene tree for the first time.
```

Comment:
Applies here and in the other PRs too, private functions (those denoted by a _ prefix) don't need public documentation. So a single `#` here is correct.
Only public methods and export variables should have a `##` comment that causes the description to be added to the online help database.